From 1a3c1d7da7b6e6dbe1ccab73e13259b624579161 Mon Sep 17 00:00:00 2001
From: Abhishek <apatgaonkar@habana.ai>
Date: Thu, 20 Jun 2024 14:10:48 +0300
Subject: [PATCH] [FSW-2704] fs1: realgin progress bits for 2 manifest design

In new design instead of separate manifest for recovery img,
there will be two manifests each having its own recovery img
and other instances of spi images. This patch aligns progress
bits accordingly.

Change-Id: Ic1b9f942336d1d97378b127ae776fea727bc2361
---

diff --git a/arch/fs1/arc/progress_bits.h b/arch/fs1/arc/progress_bits.h
index f4b6e63..f4a5cf5 100644
--- a/arch/fs1/arc/progress_bits.h
+++ b/arch/fs1/arc/progress_bits.h
@@ -9,135 +9,139 @@
 enum boot_rom_progress {
 	PROGRESS_INIT = 0,
 	PROGRESS_INIT_ARC1_ICCM_INIT_FAILED = 1,
-	PROGRESS_INIT_SPI_BOOT_DESC_EXCEEDS_MAX_SIZE = 2,
-	PROGRESS_INIT_SPI_BOOT_DESC_DMA_START = 3,
-	PROGRESS_INIT_SPI_BOOT_DESC_CRC_FAIL = 4,
-	PROGRESS_INIT_SPI_BOOT_DESC_UNALIGNED_DEST = 5,
-	PROGRESS_INIT_SPI_BOOT_DESC_INVALID_IMG_SIZE = 6,
-	PROGRESS_INIT_SPI_BOOT_DESC_UNALIGNED_IMG_SIZE = 7,
-	PROGRESS_INIT_SPI_BOOT_DESC_INVALID_FLASH_OFFSET = 8,
-	PROGRESS_INIT_SPI_BOOT_DESC_DMA_ERROR = 9,
-	PROGRESS_INIT_SPI_BOOT_DESC_DMA_FINISH = 10,
-	PROGRESS_INIT_SPI_BOOT_DESC_HDR_MAGIC_FAIL = 11,
-	PROGRESS_INIT_SPI_BOOT_DESC_INVALID_IMG_OFFSET = 12,
-	PROGRESS_INIT_SPI_BOOT_DESC_VERIFICATION_FAIL = 13,
-	PROGRESS_INIT_SPI_BOOT_DESC_INFO_SET_FAIL = 14,
-	PROGRESS_INIT_SPI_BOOT_DESC_DATA_SET_FAIL = 15,
-	PROGRESS_INIT_SPI_BOOT_DESC_INFO_GET_FAIL = 16,
-	PROGRESS_INIT_SPI_BOOT_DESC_DATA_GET_FAIL = 17,
-	PROGRESS_INIT_SPI_BOOT_DESC_FAIL = 18,
-	PROGRESS_INIT_EFUSE_INIT_FAIL = 19,
-	PROGRESS_INIT_EFUSE_SPI = 20,
-	PROGRESS_INIT_EFUSE_SPI_SLV_TIMEDOUT = 21,
-	PROGRESS_REVOKE_LOC_ASIC_EFUSE = 22,
-	PROGRESS_REVOKE_LOC_FAKE_FUSE = 23,
-	PROGRESS_KEY_LOC = 24,
-	PROGRESS_KEY_LOC_0 = 25,
-	PROGRESS_KEY_LOC_1 = 26,
-	PROGRESS_VER_LOC = 27,
-	PROGRESS_VER_LOC_0 = 28,
-	PROGRESS_VER_LOC_1 = 29,
-	PROGRESS_SLV_DIE_WAITING_ON_RECOVERY_IND = 30,
-	PROGRESS_SPI_BOOT_DESC_VERIFY_START = 31,
-	PROGRESS_SPI_BOOT_DESC_VERIFY_VERSION_FAIL = 32,
-	PROGRESS_SPI_BOOT_DESC_VERIFY_KEY_REVOKED = 33,
-	PROGRESS_SPI_BOOT_DESC_VERIFY_KEY_HASH_FAIL = 34,
-	PROGRESS_SPI_BOOT_DESC_VERIFY_AUTH_FAIL = 35,
-	PROGRESS_SPI_BOOT_DESC_VERIFY_SUCCESS = 36,
-	PROGRESS_BRINGUP_PCI_PRI = 37,
-	PROGRESS_BRINGUP_PCI_PRI_LOADADDR_INVALID = 38,
-	PROGRESS_BRINGUP_PCI_PRI_STARTADDR_INVALID = 39,
-	PROGRESS_BRINGUP_PCI_PRI_TOO_BIG = 40,
-	PROGRESS_BRINGUP_PCI_PRI_CRC_FAIL = 41,
-	PROGRESS_BRINGUP_PCI_PRI_UNALIGNED_DEST_ADDR = 42,
-	PROGRESS_BRINGUP_PCI_PRI_INVALID_IMG_SIZE = 43,
-	PROGRESS_BRINGUP_PCI_PRI_UNALIGNED_IMG_SIZE = 44,
-	PROGRESS_BRINGUP_PCI_PRI_INVALID_FLASH_OFFSET = 45,
-	PROGRESS_BRINGUP_PCI_PRI_DMA_ERROR = 46,
-	PROGRESS_BRINGUP_PCI_PRI_DMA = 47,
-	PROGRESS_BRINGUP_PCI_PRI_KEY_REVOKED = 48,
-	PROGRESS_BRINGUP_PCI_PRI_KEY_AUTH_FAIL = 49,
-	PROGRESS_BRINGUP_PCI_PRI_SECURITY = 50,
-	PROGRESS_BRINGUP_PCI_SEC = 51,
-	PROGRESS_BRINGUP_PCI_SEC_LOADADDR_INVALID = 52,
-	PROGRESS_BRINGUP_PCI_SEC_STARTADDR_INVALID = 53,
-	PROGRESS_BRINGUP_PCI_SEC_TOO_BIG = 54,
-	PROGRESS_BRINGUP_PCI_SEC_CRC_FAIL = 55,
-	PROGRESS_BRINGUP_PCI_SEC_UNALIGNED_DEST_ADDR = 56,
-	PROGRESS_BRINGUP_PCI_SEC_INVALID_IMG_SIZE = 57,
-	PROGRESS_BRINGUP_PCI_SEC_UNALIGNED_IMG_SIZE = 58,
-	PROGRESS_BRINGUP_PCI_SEC_INVALID_FLASH_OFFSET = 59,
-	PROGRESS_BRINGUP_PCI_SEC_DMA_ERROR = 60,
-	PROGRESS_BRINGUP_PCI_SEC_DMA = 61,
-	PROGRESS_BRINGUP_PCI_SEC_KEY_REVOKED = 62,
-	PROGRESS_BRINGUP_PCI_SEC_KEY_AUTH_FAIL = 63,
-	PROGRESS_BRINGUP_PCI_SEC_SECURITY = 64,
-	PROGRESS_BOOT_ROM_TPM = 65,
-	PROGRESS_BOOT_ROM_TPM_INIT_FAIL = 66,
-	PROGRESS_BOOT_ROM_TPM_STARTUP_FAIL = 67,
-	PROGRESS_BOOT_ROM_SVN_TPM_PCR_EXTEND_FAIL = 68,
-	PROGRESS_BOOT_ROM_MANIFEST_TPM_PCR_EXTEND_FAIL = 69,
-	PROGRESS_BOOT_ROM_TPM_FSM_OUT_OF_SYNC = 70,
-	PROGRESS_BOOT_ROM_TPM_FINISH = 71,
-	PROGRESS_PPBOOT_START = 72,
-	PROGRESS_PPBOOT_FINISH = 73,
-	PROGRESS_BOOT_ROM_IN_RECOVERY = 74,
-	PROGRESS_BOOT_ROM_IN_ROM_RECOVERY = 75,
-	PROGRESS_BOOT_ROM_PCIE_LINK_START = 76,
-	PROGRESS_BOOT_ROM_PCIE_LINK_INIT = 77,
-	PROGRESS_BOOT_ROM_PCIE_PHY_SRAM_INIT_TIMEDOUT = 78,
-	PROGRESS_BOOT_ROM_PCIE_PHY_PARA_STS_CLR_TIMEDOUT = 79,
-	PROGRESS_BOOT_ROM_PCIE_LINK_LIMIT_GEN1 = 80,
-	PROGRESS_BOOT_ROM_PCIE_LINK_IATU_CONFIG = 81,
-	PROGRESS_BOOT_ROM_PCIE_LINK_FINISH = 82,
-	PROGRESS_BOOT_ROM_D2D_LINK_START = 83,
-	PROGRESS_BOOT_ROM_D2D_LINK_STS_BUSY = 84,
-	PROGRESS_BOOT_ROM_D2D_LINK_INTER_DIE_SYNC = 85,
-	PROGRESS_BOOT_ROM_D2D_LINK_INIT_TIMEDOUT = 86,
-	PROGRESS_BOOT_ROM_D2D_LINK_INIT_SUCCESS = 87,
-	PROGRESS_BOOT_ROM_RECOVERY_END = 88,
-	PROGRESS_BOOT_ROM_IN_RECOVERY_IMG = 89,
-	PROGRESS_INIT_RECOVERY_IMG_BOOT_DESC_EXCEEDS_MAX_SIZE = 90,
-	PROGRESS_INIT_RECOVERY_IMG_BOOT_DESC_DMA_START = 91,
-	PROGRESS_INIT_RECOVERY_IMG_BOOT_DESC_CRC_FAIL = 92,
-	PROGRESS_INIT_RECOVERY_IMG_BOOT_DESC_UNALIGNED_DEST = 93,
-	PROGRESS_INIT_RECOVERY_IMG_BOOT_DESC_INVALID_IMG_SIZE = 94,
-	PROGRESS_INIT_RECOVERY_IMG_BOOT_DESC_UNALIGNED_IMG_SIZE = 95,
-	PROGRESS_INIT_RECOVERY_IMG_BOOT_DESC_INVALID_FLASH_OFFSET = 96,
-	PROGRESS_INIT_RECOVERY_IMG_BOOT_DESC_DMA_ERROR = 97,
-	PROGRESS_INIT_RECOVERY_IMG_BOOT_DESC_DMA_FINISH = 98,
-	PROGRESS_INIT_RECOVERY_IMG_BOOT_DESC_HDR_MAGIC_FAIL = 99,
-	PROGRESS_INIT_RECOVERY_IMG_BOOT_DESC_INVALID_IMG_OFFSET = 100,
-	PROGRESS_INIT_RECOVERY_IMG_BOOT_DESC_VERIFICATION_FAIL = 101,
-	PROGRESS_INIT_RECOVERY_IMG_BOOT_DESC_FAIL = 102,
-	PROGRESS_RECOVERY_IMG_BOOT_DESC_VERIFY_START = 103,
-	PROGRESS_RECOVERY_IMG_BOOT_DESC_VERIFY_VERSION_FAIL = 104,
-	PROGRESS_RECOVERY_IMG_BOOT_DESC_VERIFY_KEY_REVOKED = 105,
-	PROGRESS_RECOVERY_IMG_BOOT_DESC_VERIFY_KEY_HASH_FAIL = 106,
-	PROGRESS_RECOVERY_IMG_BOOT_DESC_VERIFY_AUTH_FAIL = 107,
-	PROGRESS_RECOVERY_IMG_BOOT_DESC_VERIFY_SUCCESS = 108,
-	PROGRESS_RECOVERY_IMG_IMG0 = 109,
-	PROGRESS_RECOVERY_IMG_IMG0_LOADADDR_INVALID = 110,
-	PROGRESS_BOOT_REOCVERY_IMG_IMG0_STARTADDR_INVALID = 111,
-	PROGRESS_BOOT_REOCVERY_IMG_IMG0_DMA_START = 112,
-	PROGRESS_BOOT_REOCVERY_IMG_IMG0_DMA_CRC_FAILED = 113,
-	PROGRESS_BOOT_REOCVERY_IMG_IMG0_DMA_UNALIGNED_DEST = 114,
-	PROGRESS_BOOT_REOCVERY_IMG_IMG0_DMA_INVALID_IMG_SIZE = 115,
-	PROGRESS_BOOT_REOCVERY_IMG_IMG0_DMA_UNALIGNED_IMG_SIZE = 116,
-	PROGRESS_BOOT_REOCVERY_IMG_IMG0_DMA_INVALID_FLASH_OFFSET = 117,
-	PROGRESS_BOOT_REOCVERY_IMG_IMG0_DMA_ERROR = 118,
-	PROGRESS_BOOT_REOCVERY_IMG_IMG0_DMA_FINISH = 119,
-	PROGRESS_BOOT_REOCVERY_IMG_IMG0_VERIFICATION_FAIL = 120,
-	PROGRESS_BOOT_REOCVERY_IMG_IMG0_START = 121,
-	PROGRESS_BOOT_REOCVERY_IMG_IMG0_FINISH = 122,
-	PROGRESS_BOOT_INFO_IMG_DESC_INFO_SET_START = 123,
-	PROGRESS_BOOT_INFO_IMG_DESC_INFO_SET_FINISH = 124,
-	PROGRESS_BOOT_INFO_IMG_DESC_INFO_GET_START = 125,
-	PROGRESS_BOOT_INFO_IMG_DESC_INFO_GET_FINISH = 126,
-	PROGRESS_BOOT_INFO_LFS_DISK_MOUNT_FAIL = 127,
-	PROGRESS_BOOT_INFO_LFS_FILE_OPEN_CFG_FAIL = 128,
-	PROGRESS_BOOT_INFO_LFS_FILE_COPY_PART_FAIL = 129,
-	PROGRESS_BOOT_INFO_LFS_FILE_SET_FAIL = 130,
+	PROGRESS_INIT_SPI_MANIFEST_0_EXCEEDS_MAX_SIZE = 2,
+	PROGRESS_INIT_SPI_MANIFEST_0_DMA_START = 3,
+	PROGRESS_INIT_SPI_MANIFEST_0_CRC_FAIL = 4,
+	PROGRESS_INIT_SPI_MANIFEST_0_UNALIGNED_DEST = 5,
+	PROGRESS_INIT_SPI_MANIFEST_0_INVALID_IMG_SIZE = 6,
+	PROGRESS_INIT_SPI_MANIFEST_0_UNALIGNED_IMG_SIZE = 7,
+	PROGRESS_INIT_SPI_MANIFEST_0_INVALID_FLASH_OFFSET = 8,
+	PROGRESS_INIT_SPI_MANIFEST_0_DMA_ERROR = 9,
+	PROGRESS_INIT_SPI_MANIFEST_0_DMA_FINISH = 10,
+	PROGRESS_INIT_SPI_MANIFEST_0_HDR_MAGIC_FAIL = 11,
+	PROGRESS_INIT_SPI_MANIFEST_0_INVALID_IMG_OFFSET = 12,
+	PROGRESS_INIT_SPI_MANIFEST_0_VERIFICATION_FAIL = 13,
+	PROGRESS_INIT_SPI_MANIFEST_0_INFO_SET_FAIL = 14,
+	PROGRESS_INIT_SPI_MANIFEST_0_DATA_SET_FAIL = 15,
+	PROGRESS_INIT_SPI_MANIFEST_0_INFO_GET_FAIL = 16,
+	PROGRESS_INIT_SPI_MANIFEST_0_DATA_GET_FAIL = 17,
+	PROGRESS_INIT_SPI_MANIFEST_0_FAIL = 18,
+	PROGRESS_INIT_SPI_MANIFEST_1_EXCEEDS_MAX_SIZE = 19,
+	PROGRESS_INIT_SPI_MANIFEST_1_DMA_START = 20,
+	PROGRESS_INIT_SPI_MANIFEST_1_CRC_FAIL = 21,
+	PROGRESS_INIT_SPI_MANIFEST_1_UNALIGNED_DEST = 22,
+	PROGRESS_INIT_SPI_MANIFEST_1_INVALID_IMG_SIZE = 23,
+	PROGRESS_INIT_SPI_MANIFEST_1_UNALIGNED_IMG_SIZE = 24,
+	PROGRESS_INIT_SPI_MANIFEST_1_INVALID_FLASH_OFFSET = 25,
+	PROGRESS_INIT_SPI_MANIFEST_1_DMA_ERROR = 26,
+	PROGRESS_INIT_SPI_MANIFEST_1_DMA_FINISH = 27,
+	PROGRESS_INIT_SPI_MANIFEST_1_HDR_MAGIC_FAIL = 28,
+	PROGRESS_INIT_SPI_MANIFEST_1_INVALID_IMG_OFFSET = 29,
+	PROGRESS_INIT_SPI_MANIFEST_1_VERIFICATION_FAIL = 30,
+	PROGRESS_INIT_SPI_MANIFEST_1_INFO_SET_FAIL = 31,
+	PROGRESS_INIT_SPI_MANIFEST_1_DATA_SET_FAIL = 32,
+	PROGRESS_INIT_SPI_MANIFEST_1_INFO_GET_FAIL = 33,
+	PROGRESS_INIT_SPI_MANIFEST_1_DATA_GET_FAIL = 34,
+	PROGRESS_INIT_SPI_MANIFEST_1_FAIL = 35,
+	PROGRESS_INIT_EFUSE_INIT_FAIL = 36,
+	PROGRESS_INIT_EFUSE_SPI = 37,
+	PROGRESS_INIT_EFUSE_SPI_SLV_TIMEDOUT = 38,
+	PROGRESS_REVOKE_LOC_ASIC_EFUSE = 39,
+	PROGRESS_REVOKE_LOC_FAKE_FUSE = 40, /* TODO */
+	PROGRESS_KEY_LOC = 42,
+	PROGRESS_KEY_LOC_0 = 43,
+	PROGRESS_KEY_LOC_1 = 44,
+	PROGRESS_VER_LOC = 45,
+	PROGRESS_VER_LOC_0 = 46,
+	PROGRESS_VER_LOC_1 = 47,
+	PROGRESS_SLV_DIE_WAITING_ON_RECOVERY_IND = 48,
+	PROGRESS_SPI_MANIFEST_0_VERIFY_START = 49,
+	PROGRESS_SPI_MANIFEST_0_VERIFY_VERSION_FAIL = 50,
+	PROGRESS_SPI_MANIFEST_0_VERIFY_KEY_REVOKED = 51,
+	PROGRESS_SPI_MANIFEST_0_VERIFY_KEY_HASH_FAIL = 52,
+	PROGRESS_SPI_MANIFEST_0_VERIFY_AUTH_FAIL = 53,
+	PROGRESS_SPI_MANIFEST_0_VERIFY_SUCCESS = 54,
+	PROGRESS_SPI_MANIFEST_1_VERIFY_START = 55,
+	PROGRESS_SPI_MANIFEST_1_VERIFY_VERSION_FAIL = 56,
+	PROGRESS_SPI_MANIFEST_1_VERIFY_KEY_REVOKED = 57,
+	PROGRESS_SPI_MANIFEST_1_VERIFY_KEY_HASH_FAIL = 58,
+	PROGRESS_SPI_MANIFEST_1_VERIFY_AUTH_FAIL = 59,
+	PROGRESS_SPI_MANIFEST_1_VERIFY_SUCCESS = 60,
+	PROGRESS_BRINGUP_PCI_PRI = 61,
+	PROGRESS_BRINGUP_PCI_PRI_LOADADDR_INVALID = 62,
+	PROGRESS_BRINGUP_PCI_PRI_STARTADDR_INVALID = 63,
+	PROGRESS_BRINGUP_PCI_PRI_TOO_BIG = 64,
+	PROGRESS_BRINGUP_PCI_PRI_CRC_FAIL = 65,
+	PROGRESS_BRINGUP_PCI_PRI_UNALIGNED_DEST_ADDR = 66,
+	PROGRESS_BRINGUP_PCI_PRI_INVALID_IMG_SIZE = 67,
+	PROGRESS_BRINGUP_PCI_PRI_UNALIGNED_IMG_SIZE = 68,
+	PROGRESS_BRINGUP_PCI_PRI_INVALID_FLASH_OFFSET = 69,
+	PROGRESS_BRINGUP_PCI_PRI_DMA_ERROR = 70,
+	PROGRESS_BRINGUP_PCI_PRI_DMA = 71,
+	PROGRESS_BRINGUP_PCI_PRI_KEY_REVOKED = 72,
+	PROGRESS_BRINGUP_PCI_PRI_KEY_AUTH_FAIL = 73,
+	PROGRESS_BRINGUP_PCI_PRI_SECURITY = 74,
+	PROGRESS_BRINGUP_PCI_SEC = 75,
+	PROGRESS_BRINGUP_PCI_SEC_LOADADDR_INVALID = 76,
+	PROGRESS_BRINGUP_PCI_SEC_STARTADDR_INVALID = 77,
+	PROGRESS_BRINGUP_PCI_SEC_TOO_BIG = 78,
+	PROGRESS_BRINGUP_PCI_SEC_CRC_FAIL = 79,
+	PROGRESS_BRINGUP_PCI_SEC_UNALIGNED_DEST_ADDR = 80,
+	PROGRESS_BRINGUP_PCI_SEC_INVALID_IMG_SIZE = 81,
+	PROGRESS_BRINGUP_PCI_SEC_UNALIGNED_IMG_SIZE = 82,
+	PROGRESS_BRINGUP_PCI_SEC_INVALID_FLASH_OFFSET = 83,
+	PROGRESS_BRINGUP_PCI_SEC_DMA_ERROR = 84,
+	PROGRESS_BRINGUP_PCI_SEC_DMA = 85,
+	PROGRESS_BRINGUP_PCI_SEC_KEY_REVOKED = 86,
+	PROGRESS_BRINGUP_PCI_SEC_KEY_AUTH_FAIL = 87,
+	PROGRESS_BRINGUP_PCI_SEC_SECURITY = 88,
+	PROGRESS_BOOT_ROM_TPM = 89,
+	PROGRESS_BOOT_ROM_TPM_INIT_FAIL = 90,
+	PROGRESS_BOOT_ROM_TPM_STARTUP_FAIL = 91,
+	PROGRESS_BOOT_ROM_SVN_TPM_PCR_EXTEND_FAIL, /* TODO */
+	PROGRESS_BOOT_ROM_MANIFEST_TPM_PCR_EXTEND_FAIL = 92,
+	PROGRESS_BOOT_ROM_TPM_FSM_OUT_OF_SYNC, /* TODO */
+	PROGRESS_BOOT_ROM_TPM_FINISH = 93,
+	PROGRESS_PPBOOT_START = 94,
+	PROGRESS_PPBOOT_FINISH = 95,
+	PROGRESS_BOOT_ROM_IN_RECOVERY = 96,
+	PROGRESS_BOOT_ROM_IN_ROM_RECOVERY = 97,
+	PROGRESS_BOOT_ROM_PCIE_LINK_START = 98,
+	PROGRESS_BOOT_ROM_PCIE_LINK_INIT = 99,
+	PROGRESS_BOOT_ROM_PCIE_PHY_SRAM_INIT_TIMEDOUT = 100,
+	PROGRESS_BOOT_ROM_PCIE_PHY_PARA_STS_CLR_TIMEDOUT = 101,
+	PROGRESS_BOOT_ROM_PCIE_LINK_LIMIT_GEN1 = 102,
+	PROGRESS_BOOT_ROM_PCIE_LINK_IATU_CONFIG = 103,
+	PROGRESS_BOOT_ROM_PCIE_LINK_FINISH = 104,
+	PROGRESS_BOOT_ROM_D2D_LINK_START = 105,
+	PROGRESS_BOOT_ROM_D2D_LINK_STS_BUSY = 106,
+	PROGRESS_BOOT_ROM_D2D_LINK_INTER_DIE_SYNC = 107,
+	PROGRESS_BOOT_ROM_D2D_LINK_INIT_TIMEDOUT = 108,
+	PROGRESS_BOOT_ROM_D2D_LINK_INIT_SUCCESS = 109,
+	PROGRESS_BOOT_ROM_RECOVERY_END = 110,
+	PROGRESS_BOOT_ROM_IN_RECOVERY_IMG = 111,
+	PROGRESS_RECOVERY_IMG_IMG0 = 112,
+	PROGRESS_RECOVERY_IMG_IMG0_LOADADDR_INVALID = 113,
+	PROGRESS_BOOT_RECOVERY_IMG_IMG0_STARTADDR_INVALID = 114,
+	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_START = 115,
+	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_CRC_FAILED = 116,
+	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_UNALIGNED_DEST = 117,
+	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_INVALID_IMG_SIZE = 118,
+	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_UNALIGNED_IMG_SIZE = 119,
+	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_INVALID_FLASH_OFFSET = 120,
+	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_ERROR = 121,
+	PROGRESS_BOOT_RECOVERY_IMG_IMG0_DMA_FINISH = 122,
+	PROGRESS_BOOT_RECOVERY_IMG_IMG0_VERIFICATION_FAIL = 123,
+	PROGRESS_BOOT_RECOVERY_IMG_IMG0_START = 124,
+	PROGRESS_BOOT_RECOVERY_IMG_IMG0_FINISH = 125,
+	PROGRESS_BOOT_INFO_IMG_DESC_INFO_SET_START = 140,
+	PROGRESS_BOOT_INFO_IMG_DESC_INFO_SET_FINISH = 141,
+	PROGRESS_BOOT_INFO_IMG_DESC_INFO_GET_START = 142,
+	PROGRESS_BOOT_INFO_IMG_DESC_INFO_GET_FINISH = 143,
+	PROGRESS_BOOT_INFO_LFS_DISK_MOUNT_FAIL = 144,
+	PROGRESS_BOOT_INFO_LFS_FILE_OPEN_CFG_FAIL = 145,
+	PROGRESS_BOOT_INFO_LFS_FILE_COPY_PART_FAIL = 146,
+	PROGRESS_BOOT_INFO_LFS_FILE_SET_FAIL = 147,
 };
 
 #endif /* _PROGRESS_BITS_H_ */
